src/main/java/com/ibm/crypto/plus/provider/ock/Digest.java

```diff
@@ -51,7 +51,7 @@ public final class Digest implements Cloneable {
     static private int runtimeContextNum[];
 
     class ConcurrentLinkedQueueLong extends ConcurrentLinkedQueue<Long> {
-        /* empty */
+        private static final long serialVersionUID = 196745693267521676L;
```

Comment:
Does it necessnary to setup a serialVersionUID for Digest class? I am not sure if Digest object will be serialized and deserialized.

Reply:
This is not the `Digest` that is serializable, but the inner class. It inherits it from the `ConcurrentLinkedQueue`. We maybe could eventually remove this inner class, but it's not as straightforward.

Reply:
Right, its come from the ConcurrentLinkedQueue.